extensions/wikia/Lightbox/scripts/Lightbox.js

```diff
@@ -76,13 +76,6 @@
 			// cache re-used DOM elements and templates for this modal instance
 			Lightbox.cacheDOM();
 
-			// Init ads in Lightbox
-			if ($('#MODAL_RECTANGLE').length && Lightbox.ads.showAds()) {
-				Lightbox.openModal.lightbox.addClass('show-ads');
```

Comment:
Maybe we also need to remove: https://github.com/Wikia/app/blob/ADEN-2983-unusedSlots/extensions/wikia/Lightbox/styles/Lightbox.scss#L402-L435 ?